package.json

```diff
@@ -86,5 +92,9 @@
   "peerDependencies": {
     "react": ">=18.2.0",
     "react-dom": ">=18.2.0"
+  },
+  "lint-staged": {
```

Comment:
Should be something like this: 
```json
"lint-staged": {
    "*.{ts,tsx,js,jsx}": "eslint --cache --fix",
    "*.{ts,tsx,js,jsx,css,less,md,html}": "prettier --write"
  }
```
since we want eslint to check `js,jsx` files too and prettier to format `ts,tsx,jsx,less,html` files as well. Also `.eslintcache` file should be added to `.gitignore`.